fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java

```diff
@@ -766,6 +767,17 @@ public TMasterOpResult proxyExecute(TMasterOpRequest request) {
             ctx.setCurrentRoleIds(new HashSet<>());
         }
 
+        if (request.isSetSession_id()) {
+            ctx.setSessionId(UUID.fromString(request.getSession_id()));
+        } else {
+            TMasterOpResult result = new TMasterOpResult();
```

Comment:
When a cluster is upgraded to the new version, the new leader will accept a request without session_id.
We need to handle this request correctly rather than returning an error packet. This will make user feel something bad happens and rollback the new version.

Reply:
> When a cluster is upgraded to the new version, the new leader will accept a request without session_id. We need to handle this request correctly rather than returning an error packet. This will make user feel something bad happens and rollback the new version.

make sense. I think this error can be avoided. I've fixed it and add some comments here. @imay